tests/param_shift_dev.py

```diff
@@ -27,9 +27,11 @@ class ParamShiftDerivativesDevice(qml.devices.DefaultQubit):
     name = "param_shift.qubit"
 
     def preprocess(self, execution_config=qml.devices.DefaultExecutionConfig):
-        if config.gradient_method in {"device", "parameter-shift"}:
```

Comment:
nice find. curious that this didn't cause issues

Reply:
Only now does it have integration tests with `qml.execute`. Before it was just getting used with `DeviceDerivatives` and `DeviceJacobianProducts`.